### DISCUSSION TITLE
Pull-to-refresh on mobile temporarily hides all notifications

### DISCUSSION THREAD

**Discussion by ZevEisenberg** [community/community#69693] [Mobile]
When I load the app, it shows my notifications. When I pull to refresh. They all disappear. If I tap the Notifications tab, or leave and come back to that tab, they come back. Reproducible after restart. This started in the latest TestFlight beta or two. (But it replaces a previous issue I was seeing where the loading spinner at the upper right of the Notifications tab was never stopping.)

https://github.com/community/community/assets/464574/c1be8c8b-9a8b-40c8-9853-e8975960ff8b

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.134.0 (111895692)
Device: iPad Pro 11-inch (iOS 17.0.3)
</details>

**Comment by esahmed21** [comment]
Hi @ZevEisenberg 👋 

Thank you for reaching out! That's absolutely a bug that shouldn't happen, i was also able to reproduce. I will create an issue internally for the team to prioritise and fix it!
